vm/actor/src/builtin/market/mod.rs

```diff
@@ -279,7 +278,7 @@ impl Actor {
                     .pending_deals
                     .as_ref()
                     .unwrap()
-                    .contains_key(&pcid.to_bytes())
+                    .has(&pcid.to_bytes())
```

Comment:
optional: I know not changed in this, but the `has` function could use the `contains_key` method on the Hamt that was added after the method was added on the `Set` (functionally the same)